Examples/KlaviyoSwiftExamples/SPMExample/SPMExample.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/klaviyo/klaviyo-swift-sdk",
       "state" : {
-        "revision" : "bcfb37dac6d424a20ae8e77bbe5aafdb66e8228f",
```

Comment:
updated to master since there were changes that were not present in earlier released version and this was causing a compiler issue.